src/components/Approval/Approval.tsx

```diff
@@ -53,6 +49,8 @@ export const Approval = () => {
   const history = useHistory()
   const approvalInterval: { current: NodeJS.Timeout | undefined } = useRef()
   const [approvalTxId, setApprovalTxId] = useState<string>()
+  const [isExactAllowance, setIsExactAllowance] = useState<boolean>(false)
+  const [buildApprovalTxInput, setBuildApprovalTxInput] = useState<evm.BuildCustomTxInput>()
```

Comment:
There is no `buildApprovalTx` method, I reckon this should be `buildCustomTxArgs` / `buildCustomTxInput` to conform to the arg `approve()` takes (which is also used as generic here)? Unless the intent is to make it explicit this is for approvals explicitly, in which case we can simply ditch the `build` terminology to keep things less confusing.

https://github.com/shapeshift/web/blob/4d6904e8d529ee68e21bb85c4d961b1566c0542c/src/components/Trade/hooks/useSwapper/useSwapper.tsx#L121-L122